libraries/AP_HAL_ChibiOS/hwdef/OrqaF405Pro/README.md

```diff
@@ -0,0 +1,83 @@
+# Orqa FC 3030 F405 Flight Controller
+
+The Orqa FC 3030 F405 is a flight controller produced by [Orqa](https://enterprise.orqafpv.com/).
+
+## Features
+
+ - MCU - STM32F405 32-bit processor
+ - IMU - MPU6000
+ - Barometer - DPS310
+ - OSD - AT7456E
+ - Onboard Flash: 128Mbit
+ - 4x UARTs
+ - 8x PWM Outputs (8 Motor Output)
+ - Battery input voltage: 2S-6S
+ - BEC 5V 2A
+ - BEC 10V 2A
+
+## Pinout
+
+![Orqa FC 3030 F405 Board](OrqaFC3030F405_Board.png "Orqa FC 3030 F405")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (RX/GHST, DMA-enabled)
+ - SERIAL3 -> UART3 (GPS, DMA-enabled)
+ - SERIAL5 -> UART5 (ESC Telemetry)
+ - SERIAL6 -> UART6 (Sik/MAVLink)
+
+## RC Input
+
+RC input is configured for GHST by default on the T1 (UART1_TX) pin. To support other RC protocols set SERIAL1_OPTIONS to 0. This will support all serial RC protocols except SBUS. For PPM and SBUS support on UART1_RX set BRD_ALT_CONFIG to 1.
+  
+## OSD Support
+
+The Orqa FC 3030 F405 supports OSD using OSD_TYPE 1 (MAX7456 driver) or OSD_TYPE 3 if using DJI OSD
+
+## PWM Output
+
+The Orqa FC 3030 F405 supports up to 8 PWM outputs in the JST-GH ESC and MFC connectors.
+
+The PWM is in 4 groups:
+
+ - PWM 1,4 in group1
+ - PWM 2-3 in group2
+ - PWM 5-6 in group3
+ - PWM 7-8 in group4
+
+Channels within the same group need to use the same output rate. If
+any channel in a group uses DShot then all channels in the group need
+to use DShot. Channels 1-4 support bi-directional dshot.
+
+## Battery Monitoring
+
+The board has a builting voltage and current sensor. The current
+sensor can read up to 130 Amps. The voltage sensor can handle up to 6S
+LiPo batteries.
+
+The correct battery setting parameters are:
+
+ - BATT_MONITOR 4
+ - BATT_VOLT_PIN 13
+ - BATT_CURR_PIN 12
+ - BATT_VOLT_MULT 8.3
+ - BATT_AMP_PERVLT 92.6
+
```

Comment:
should have a list of ad/gpio pins and a section on vtx pwr control...the +10v pin, right? there are four gpio output pins in pinout that I cant find defined in hwdef

Reply:
"GPIO" are actually Motor 5-8, can obviously be configured as GPIO as well. Added VTX power section.

Reply:
needs to be documented...see above